env/src/test/java/com/sap/cloud/security/config/json/DefaultJsonObjectTest.java

```diff
@@ -3,8 +3,11 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-package com.sap.cloud.security.json;
+package com.sap.cloud.security.config.json;
```

Comment:
this is not the right package, you need to move it outside config package

Reply:
great finding. should be corrected now.